Gemfile

```diff
@@ -1,5 +1,5 @@
 source 'https://rubygems.org'
 
 gem 'benchmark-ips', '>= 2.0'
-
+gem 'activesupport'
```

Comment:
@nirvdrum Agree.
As another solution, I can just add implementation of `try!` in benchmark file